content-elements/tce-embed/edit/index.vue

```diff
@@ -43,11 +44,13 @@ export default {
     },
     showPlaceholder() {
       return !this.element.data.url;
+    },
+    id() {
+      return this.element._cid || this.element.id;
     }
   },
-  created() {
-    if (this.element.data.height) return;
-    this.$emit('save', { height: 300, url: '' });
+  mounted() {
+    this.$elementBus.on('save', data => this.$emit('save', data));
```

Comment:
Why is he listening to the toolbar save via bus ? (both should be able to update store by emitting regular save event)